library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2160,6 +2160,108 @@ void libspdm_set_last_spdm_error_struct(void *spdm_context, libspdm_error_struct
                      last_spdm_error, sizeof(libspdm_error_struct_t));
 }
 
+#if LIBSPDM_FIPS_MODE
+/**
+ * Initialize an libspdm_crypto_context.
+ *
+ * @param  crypto_context       A pointer to the crypto_context.
+ *
+ * @retval RETURN_SUCCESS       context is initialized.
+ * @retval RETURN_DEVICE_ERROR  context initialization failed.
+ */
+libspdm_return_t libspdm_init_crypto_context(void *crypto_context)
+{
+    libspdm_crypto_context *context;
+    LIBSPDM_ASSERT(crypto_context != NULL);
+
+    context = crypto_context;
+
+    /*No tested for every uesd algo*/
+    context->tested_algo = 0;
+    /*self_test reuslt is false for every uesd algo*/
+    context->self_test_result = 0;
+
+    return LIBSPDM_STATUS_SUCCESS;
+}
+
+/**
+ * Return the size in bytes of the crypto_context.
+ *
+ * @return the size in bytes of the crypto_context.
+ **/
+size_t libspdm_get_crypto_context_size(void)
+{
+    size_t size;
+
+    size = sizeof(libspdm_crypto_context);
+    return size;
+}
+
+/**
+ * import crypto_context to spdm_context;
+ *
+ * @param[in,out]  spdm_context         A pointer to the spdm_context.
+ * @param[in]      crypto_context       A pointer to the crypto_context.
+ * @param[in]      crypto_context_size  The size of crypto_context.
+ *
+ * @retval true   import crypto_context successful.
+ * @retval false  spdm_context or crypto_context is null.
+ */
+bool libspdm_import_crypto_context_to_spdm_context(void *spdm_context, void *crypto_context,
+                                                   size_t crypto_context_size)
```

Comment:
I'm curious, why is parameter crypto_context_size needed here?  If it's needed here for safety, for consistency, why not also have a parameter "spdm_context_size"?